synfig-studio/src/gui/app.cpp

```diff
@@ -35,137 +35,123 @@
 #	include <config.h>
 #endif
 
-#include <fstream>
 #include <iostream>
-#include <locale>
-#include <cstring>
+#include <fstream>
```

Comment:
Whoops! I noticed I sort it wrong here :P 

@ice0 did you change anything on this PR? Or i can fix it here and force-push it again?

Reply:
No, I just rebased it for merge, but there were problems with Appveyor yesterday, so it still hasn't merged.

Reply:
Ok. So I force pushed to sort it right